workspaces/lightspeed/plugins/lightspeed-backend/README.md

```diff
@@ -33,6 +33,9 @@ Add the following lightspeed configurations into your `app-config.yaml` file:
 lightspeed:
   servicePort: <portNumber> # Optional - Change the LS service port nubmer. Defaults to 8080.
   systemPrompt: <system prompt> # Optional - Override the default system prompt.
+  mcpServers: # Optional - only one mcp server is currently supported
+    name: <mcp server name> # must match the name configured in LCS
+    token: ${MCP_TOKEN}
```

Comment:
```suggestion
  - name: <mcp server name> # must match the name configured in LCS
    token: ${MCP_TOKEN}
```

array

Reply:
Not sure if you read my msg, but is it possible to do this in runtime on Backstage? Seems to be working locally on lightspeed backend:

plugins/lightspeed-backend/src/service/router.ts
```
const mcpTokenENV = "MCP_TOKEN" (read from config lightspeed.mcpServers[0].tokenENV)

const mcpToken2 = process.env[mcpTokenENV];

if (mcpToken2) {
  console.log(`The MCP_TOKEN is: ${mcpToken2}`);
} else {
  console.error('MCP_TOKEN environment variable not set.');
}
```

this way, config can be without token:
```
lightspeed:
  mcpServers:
  - name: mcp::backstage
    tokenENV: MCP_TOKEN
```

output from log:
```
2025-10-16T20:37:22.413Z lightspeed info /v1/query receives call from user: user:development/guest 
The MCP_TOKEN is: my-mcp-token
```